cypress/integration/channel/message_spec.js

```diff
@@ -128,4 +128,15 @@ describe('Message', () => {
                 next().should('have.text', '@channel').should('not.have.text', '.');
         });
     });
+
```

Comment:
Please document the steps according to the file header

Reply:
@reflog I've corrected my documentation for the cypress test